java/ql/lib/ext/java.util.stream.model.yml

```diff
@@ -92,4 +92,7 @@ extensions:
       pack: codeql/java-all
       extensible: neutralModel
     data:
+      - ["java.util.stream", "Collectors", "joining", "(CharSequence)", "manual"]
```

Comment:
This seems like it ought to be taint preserving, though I'm not sure how it would fit in with the special-case handling of the collect api

Reply:
I've added a summary model for this for flow from `Argument[0] -> ReturnValue` in  https://github.com/github/codeql/pull/11779/commits/181a711f045012f91f06565e77ba43a633304f97. It does feel somewhat incomplete without having the `collect` api handled by MaD as well though 🤔

Reply:
`Collector` APIs are covered [here](https://github.com/github/codeql/blob/ce06df315236eb011ffa269ef5a88e2527af4424/java/ql/lib/semmle/code/java/frameworks/Stream.qll#L44). As you said, this model isn't going to do much without another model that reads taint from the collector, which doesn't currently exist. I _think_ it won't hurt having it either, so we could keep it — with the risk of causing confusion to a future reader that wonders why we modeled this incompletely in MaD.

Reply:
> So it's probably best to remove it.

I've removed it in https://github.com/github/codeql/pull/11779/commits/ac064ac2a78aae7c7096a519949e30765f3c58ad